docs/php/versioning.md

```diff
@@ -57,7 +57,7 @@ $result2 = yield $yourActivity->activityB($result1);
 return $result2;
 ```
 
-When `Workflow::getVersion()` is run for the new Workflow execution, it records a marker in the Workflow
+When `Workflow::getVersion()` is run for the new Workflow Execution, it records a marker in the Workflow
 history so that all future calls to `GetVersion` for this change Id--`Step 1` in the example--on this
 Workflow Execution will always return the given version number, which is `1` in the example.
```

Comment:
```suggestion
When `Workflow::getVersion()` is run for the new Workflow Execution, it records a marker in the Workflow history so that all future calls to `GetVersion` for this change Id—`Step 1` in the example—on this Workflow Execution will always return the given version number, which is `1` in the example.
```